salt/journal/config/etc-nginx-sites-available-journal.conf

```diff
@@ -325,8 +325,30 @@ server {
     }
     {% endif -%}
 
+    {% if pillar.journal.privacy-policy %}
+    # Proxy /privacy-policy pages to pubpub
+    location ~ ^/privacy-policy[/]* {
+        proxy_pass {{ pillar.journal.privacy_notice_url }};
+        proxy_ssl_server_name on;
+        proxy_connect_timeout 10s;
+        proxy_read_timeout 10s;
+        proxy_set_header X-PubPubClient elifesciences;
+    }
+    {% endif -%}
+
+     {% if pillar.journal.cookie-declaration %}
```

Comment:
Should this be `cookie_declaration_url`? Also indentation looks a little off.

Reply:
Thanks, I've corrected it and removed the empty space